updating/migrating-to-multi-payload.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="migrating-clusters-to-multi-payload"]
+= Migrating to a cluster with multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can migrate your current cluster with single-architecture compute machines to a cluster with multi-architecture compute machines by updating to a multi-arch manifest-listed payload. This allows you to add mixed architecture compute nodes to your cluster. 
+
+For information about configuring your multi-architecture compute machines, see _Configuring multi-architecture compute machines on an {product-title} cluster_.
+
+[IMPORTANT]
+====
+Once a cluster has transitioned to using a multi-architecture payload, it can no longer return back to cluster with single-architecture compute machines or accept a single-architecture upgrade payload. 
```

Comment:
```suggestion
Once a cluster has transitioned to using a multi-architecture payload, it can no longer return back to a cluster with single-architecture compute machines or accept a single-architecture upgrade payload. 
```